rdkit/Chem/Descriptors.py

```diff
@@ -200,27 +202,39 @@ def MinAbsPartialCharge(mol, force=False):
 
 
 def _FingerprintDensity(mol, func, *args, **kwargs):
-  fp = func(*((mol, ) + args), **kwargs)
+  aname = f'_{func.__name__}_{args}'
+  if hasattr(mol, aname):
+    fp = getattr(mol, aname)
```

Comment:
Perhaps use a computed property as opposed to getattr?

Reply:
You'd have to convert the fingerprint to a string in order to be able to call `SetProp()` with it from Python.
We could do that by pickling/depickling the fingerprint, but seems like extreme overkill

Reply:
Indeed, we should perhaps expose the other POD style data to python at some point.